### ISSUE TITLE
Nice work!Do you have a pretrained_model?

### ISSUE THREAD

**Issue by aguang1201** [sirius-ai/MobileFaceNet_TF#3]

**Comment by xsr-ai** [comment]
sorry that, now i can't.

**Comment by chenkingwen** [comment]
@xsr-ai , I used MS1M to train the net, but the accuracy is always 0. I added FC to the last layer for using softmax to train the net, but the accuracy is a little shocking, The accuracy on LFW reached 97% but no better. How can I solve this problem? Thank you

**Comment by xsr-ai** [comment]
Accuracy always zero is strange，maybe it's a bug in my code, i will try to solve that. for your second issue, in my test result, loss function of softmax can't reached better acc, so why you use softamax? But if you want to get better acc than 97%, hyperparams is the key, you can try and test.

**Comment by sirius-ai** [comment]
pretrained model will be provide later.
close, when need reopen.
